app/services/testing_ground/duplicator.rb

```diff
@@ -0,0 +1,35 @@
+class TestingGround::Duplicator
```

Comment:
This class feels a *lot* like [`TestingGround::SaveAs`](https://github.com/quintel/etmoses/blob/master/app/services/testing_ground/save_as.rb) to me. Did you see that class? 😀